e2e/rbd.go

```diff
@@ -1212,6 +1212,50 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create rbd clones in different pool", func() {
+				// snapshot beta is only supported from v1.17+
+				if k8sVersionGreaterEquals(f.ClientSet, 1, 17) {
```

Comment:
Check the inverse, and use `e2e.Skip()` to continue with the next test. That makes it easier to drop checks like this for old/unsupported Kubernetes versions.

Reply:
Sounds good.